pkg/automation/automation.go

```diff
@@ -379,6 +386,7 @@ func NeedToWaitForAirgapApp() (bool, error) {
 }
 
 func getAirgapData(clientset kubernetes.Interface, license *kotsv1beta1.License) (map[string][]byte, error) {
+
```

Comment:
can we keep the current code format of not having an empty line at the beginning of the function?